pubspec.lock

```diff
@@ -0,0 +1,365 @@
+# Generated by pub
```

Comment:
we almost certainly want to exclude this!

Reply:
Done.